product_docs/docs/biganimal/release/getting_started/creating_a_cluster/index.mdx

```diff
@@ -130,7 +130,21 @@ Change the default database backup retention period of 30 days using the **Reten
 
 BigAnimal automatically deletes backups older than the retention period.
 
-### **Connections** 
+### Maintenance
+
+As a Project Owner or Project Editor, you have the ability to set/edit maintenance windows for your clusters in order to ensure the stability and security of your databases. By setting a maintenance window, you can specify a 60-minute time frame in which maintenance upgrades will occur for each cluster. If you do not set a window, BigAnimal will assign a random maintenance window for you, which may not be ideal for your needs. 
+
+1.  **Enable** the Custom Maintenance Window option. This gives you the ability to set your own maintenance window, rather than relying on a randomly assigned window.
+    
+1.  Now you see a drop-down menu where you can select the **day** and **start/end time** for your maintenance window. Choose the weekly time frame during which you want your maintenance to occur based on your needs. It is recommended that you choose a time that is least disruptive to your business operations.
+
+    !!! Note
+    The 60-minute maintenance window specifies the start time, not necessarily the end time, of maintenance events.
+    !!!
+
+BigAnimal shows a "Maintenance in progress" message (with a blue background) on the Cluster List 4 hours prior to the scheduled maintenance time. This notification reminds you to prepare for the upcoming maintenance window, and to make any necessary preparations, such as saving your work and closing any open connections.
```

Comment:
I would say `displays` instead of `shows` because it's more precise. I think `Maintenance in progress` should be `Scheduled maintenance` and in bold instead of quotations because it's a UI element. And, I would delete `(with a blue background)` because I don't think it's necessary to mention.